helm/vertical-pod-autoscaler-app/Chart.yaml

```diff
@@ -4,14 +4,19 @@ description: A Helm chart for Kubernetes Vertical Pod Autoscaler
 icon: https://s.giantswarm.io/app-icons/k8s-initiator/1/dark.svg
 home: https://github.com/giantswarm/vertical-pod-autoscaler-app
 maintainers:
-- name: sudermanjr
+  - name: giantswarm/team-atlas
+    email: team-atlas@giantswarm.io
 name: vertical-pod-autoscaler-app
 sources:
-- https://github.com/kubernetes/autoscaler
+- https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler
+dependencies:
+  - name: vertical-pod-autoscaler
+    version: 5.3.0
+    repository: https://cowboysysop.github.io/charts/
 type: application
-version: [[ .Version ]]
```

Comment:
Can we remove this placeholder now, with latest ABS?

Reply:
Yes you're right !